user_manual/groupware/sync_ios.rst

```diff
@@ -39,6 +39,16 @@ You should now find your contacts in the address book of your iPhone.
 .. note:: Beginning with iOS 12 an SSL encryption is necessary. Therefore do **not** disable **SSL**
   (For this reason a certificate is required at your domain, https://letsencrypt.org/ will do).
 
+The usual link syntax looks like this:
+
+https://<fqdn-of-server>/remote.php/dav/calendars/<username>/<calendar-name>/
+
+iOS is a little bit special and usually requires the following syntax:
+
+https://<fqdn-of-server>/remote.php/dav/principals/users/<username>/
+
+Also additionally, in order to connect NextCloud calendar or contacts on iOS you need first to generate an app password as follows:
```

Comment:
For this we could take inspiration from https://docs.nextcloud.com/server/latest/user_manual/en/groupware/sync_osx.html#setup-your-accounts and adjust "Enter your user name and password." above